File: internal/domain/reminder.go
package domain

import "time"

// PaymentReminder represents a payment reminder sent for an overdue invoice.
type PaymentReminder struct {
	ID             int64
	InvoiceID      int64
	ReminderNumber int
	SentAt         time.Time
	SentTo         string
	Subject        string
	BodyPreview    string
	CreatedAt      time.Time
}

File: rust/crates/zfaktury-domain/src/reminder.rs
use chrono::NaiveDateTime;

/// A payment reminder sent for an overdue invoice.
#[derive(Debug, Clone)]
pub struct PaymentReminder {
    pub id: i64,
    pub invoice_id: i64,
    pub reminder_number: i32,
    pub sent_at: NaiveDateTime,
    pub sent_to: String,
    pub subject: String,
    pub body_preview: String,
    pub created_at: NaiveDateTime,
}
